plugins/inputs/internet_speed/internet_speed.go

```diff
@@ -33,23 +43,47 @@ func (is *InternetSpeed) SampleConfig() string {
 
 const measurement = "internet_speed"
 
+var serverCache *speedtest.Server
+
```

Comment:
Why not adding this to the `InternetSpeed` struct?

Reply:
Good point